msteams-platform/messaging-extensions/how-to/link-unfurling.md

```diff
@@ -10,16 +10,38 @@ ms.author: anclear
 
 [!include[v4-to-v3-SDK-pointer](~/includes/v4-to-v3-pointer-me.md)]
 
-This document guides you on how to add link unfurling to your app manifest using App studio and manually. With link unfurling, your app can register to receive an `invoke` activity when URLs with a particular domain are pasted into the compose message area. The `invoke` contains the full URL that was pasted into the compose message area, and you can respond with a card that the user can unfurl, providing additional information or actions. This works similar to a search command with the URL serving as the search term.
+In link unfurling, your app can register to receive an invoke activity when URLs with a particular domain are pasted into the compose message area. The invoke contains the full URL and you can respond to it with a card. User can unfurl the link using message extension with additional information to get an enhanced experience. This works similar to a search command with the URL serving as the search term. You can add link unfurling to your app manifest using the developer portal or manually.</br> Now link unfurling is supported for both mobile and desktop.
 
-> [!NOTE]
->
-> * Currently, link unfurling is not supported on Mobile clients.
-> * The link unfurling result is cached for 30 minutes.
+## Advantages of link unfurling 
+
+* Link unfurling provides preview of the URL
+* You can get the specfic URL unfurl in the compose message box
+
+>[!NOTE]
+>The link unfurling result is cached for 30 minutes.</br>
+>Link unfurling on mobile works only for installed apps in Microsoft 365
+
+The following images displays the experience of link unfurling on mobile and desktop:
+
+# [Mobile](#tab/Samplemobileapp)
 
-The Azure DevOps message extension uses link unfurling to look for URLs pasted into the compose message area pointing to a work item. In the following image, a user has pasted a URL for a work item in Azure DevOps, which the message extension has resolved into a card:
+  Link unfurling on mobile using the following steps:
 
-:::image type="content" source="~/assets/images/compose-extensions/messagingextensions_linkunfurling.png" alt-text="Example of link unfurling":::
+   1. User has pasted an URL in the compose message area in mobile
+   1. You can see the link unfurling in the compose message area 
+   
+   :::image type="content" source="~/assets/images/Teams-link-unfurling/mobile-link-unfurl.png" alt-text="mobile microsoft" border="true":::
+
+# [Desktop](#tab/Sampledesktop)
+
+  Link unfurling on desktop using the following steps:
+
+   1. User has pasted an URL in the compose message area in desktop
```

Comment:
Write the steps correctly. Please follow correct step- by step instruction convention.